easybuild/easyconfigs/c/CAVIAR/CAVIAR-20180822-intel-2018a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'CAVIAR'
+version = '20180822'
+
+homepage = "https://github.com/fhormoz/caviar"
+description = """CAusal Variants Identication in Associated Regions. A statistical framework
+ that quantifies the probability of each variant to be causal while allowing an arbitrary
+ number of causal variants."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+# No official release tarballs available
+# Sources obtained by:
+# git clone https://github.com/fhormoz/caviar.git
+# git archive --format=tar -o CAVIAR-20180822.tar --prefix=CAVIAR/ master
+# gzip CAVIAR-20180822.tar
+sources = ['%(name)s-%(version)s.tar.gz']
```

Comment:
@verdurin Why not download via commit ID & rename on download (cfr. https://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources)?